llvm/lib/MC/MCObjectStreamer.cpp

```diff
@@ -467,20 +467,22 @@ void MCObjectStreamer::emitDwarfAdvanceLineAddr(int64_t LineDelta,
 }
 
 void MCObjectStreamer::emitDwarfLineEndEntry(MCSection *Section,
-                                             MCSymbol *LastLabel) {
+                                             MCSymbol *LastLabel,
+                                             MCSymbol *EndLabel) {
   // Emit a DW_LNE_end_sequence for the end of the section.
   // Use the section end label to compute the address delta and use INT64_MAX
   // as the line delta which is the signal that this is actually a
   // DW_LNE_end_sequence.
-  MCSymbol *SectionEnd = endSection(Section);
+  if (!EndLabel)
```

Comment:
The comment should be updated to state what a non-null EndLabel does.

Reply:
Updated